autoload/utils/exec/job.vim

```diff
@@ -121,16 +121,13 @@ function! utils#exec#job#stop() abort
         return
     endif
     let l:job = s:cmake4vim_job['job']
-    try
-        if has('nvim')
-            call jobstop(l:job)
-        else
-            call job_stop(l:job)
-        endif
-    catch
-    endtry
+    if has('nvim')
+        call jobstop(l:job)
+    else
+        call job_stop(l:job)
+    endif
+    let s:cmake4vim_jobs_pool = []
     call s:createQuickFix()
-    call s:closeBuffer()
```

Comment:
Actually I'd like this to stay, this fixes a bug with `fzf`.

If a `job` finished while a `fzf` window is open, `cmake4vim_execute` buffer does not close (because `fzf` has weird buffers), and you can't start another `job` unless you remove it manually. 

This line is helpful because you don't have to call `bwipe cmake4vim_execute`, you can instead just `call utils#exec#job#stop()`.

Reply:
I didn't change the behavior. The call of `s:closeBuffer()` function was moved to `s:createQuickFix()`, so I changed nothing.